src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -213,239 +213,240 @@
-                                    },
-                                  }"
-                                >
-                                  {{ name }}
-                                </RouterLink>
-                              </ConnectionCard>
-                            </template>
-                          </ConnectionGroup>
-                        </DataCollection>
-                      </template>
+                                {{ name }}
+                              </RouterLink>
+                            </ConnectionCard>
+                          </template>
+                        </ConnectionGroup>
+                      </DataCollection>
                     </template>
                   </template>
-                  <template v-else>
-                    <EmptyBlock />
-                  </template>
-                </ConnectionTraffic>
-              </div>
-            </KCard>
+                </template>
+                <template v-else>
+                  <EmptyBlock />
+                </template>
+              </ConnectionTraffic>
+            </div>
+          </KCard>
 
-            <RouterView
-              v-slot="child"
+          <RouterView
+            v-slot="child"
+          >
+            <SummaryView
+              v-if="child.route.name !== route.name"
+              width="670px"
+              @close="function (_e) {
+                route.replace({
+                  name: 'data-plane-detail-view',
+                  params: {
+                    mesh: route.params.mesh,
+                    dataPlane: route.params.dataPlane,
+                  },
+                  query: {
+                    inactive: route.params.inactive ? null : undefined,
+                  },
+                })
+
+              }"
             >
-              <SummaryView
-                v-if="child.route.name !== route.name"
-                width="670px"
-                @close="function (_e) {
-                  route.replace({
-                    name: 'data-plane-detail-view',
-                    params: {
-                      mesh: route.params.mesh,
-                      dataPlane: route.params.dataPlane,
-                    },
-                    query: {
-                      inactive: route.params.inactive ? null : undefined,
-                    },
-                  })
-
-                }"
-              >
-                <component
-                  :is="child.Component"
-                  :data="(child.route.name as string).includes('-inbound-') ? inbounds : traffic?.outbounds || {}"
-                />
-              </SummaryView>
-            </RouterView>
+              <component
+                :is="child.Component"
+                :data="(child.route.name as string).includes('-inbound-') ? props.data.dataplane.networking.inbounds : traffic?.outbounds || {}"
+                :dataplane-overview="props.data"
```

Comment:
As a consequence of moving the template I also tweaked this line, and added the `:dataplane-overview` which I then use in the sub views.